kernel/kata_config_version

```diff
@@ -1 +1 @@
-83
+84
```

Comment:
need to rebase, and reincrement.

Reply:
(heads up, too, that there's another PR I hope lands in the next day that'll also increment...